src/kong/clean.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Helper script to help clean up the auto generated src/cms/extensions/documentation/documentation/1.0.0/full_documentation.json
+ * for the Kong gwa client.
+ * 
+ * Mainly used to remove non-publicly used Strapi endpoints in the OpenAPI spec
```

Comment:
It seems like there should be a way to just mark these for no documentation in strapi. This is fine for now though.